internal/utils/ocpbuild/utils.go

```diff
@@ -12,11 +12,11 @@ const (
 	HashAnnotation = "kmm.node.kubernetes.io/last-hash"
 )
 
-func GetBuildAnnotations(hash uint64) map[string]string {
+func GetOCPBuildAnnotations(hash uint64) map[string]string {
```

Comment:
Do we also need to add OCP to the function names based on the fact there are in the ocpbuildutils package?
Same question for all other functions in this file.

Reply:
I would change it to OCP and keep it, makes code clearer